backend/src/main/java/com/javahelp/backend/data/ICRUDGateway.java

```diff
@@ -0,0 +1,47 @@
+package com.javahelp.backend.data;
+
+/**
+ * Interface specifying generic CRUD operations for the provided type.
+ * <p></p>
```

Comment:
Just a little question, what does ``<p></p>`` do, though?

Reply:
It's just a new paragraph tag